test/e2e/storage/testsuites/multivolume.go

```diff
@@ -329,6 +330,106 @@ func (t *multiVolumeTestSuite) DefineTests(driver storageframework.TestDriver, p
 			l.config.ClientNodeSelection, resource.Pvc, numPods, true /* sameNode */, false /* readOnly */)
 	})
 
+	// This tests below configuration:
+	// [pod1]           [pod2]
+	// [        node1        ]
+	//   |                 |     <- same volume mode
+	// [volume1]   ->  [restored volume1 snapshot]
+	ginkgo.It("should concurrently access the volume and restored snapshot from pods on the same node [LinuxOnly][Feature:VolumeSnapshotDataSource][Feature:XFS]", func() {
```

Comment:
cc @jingxu97 to see if we can remove LinuxOnly

Reply:
The new tests uses part of snapshot/cloning tests in `provisioning.go`:
* [`should provision storage with snapshot data source [Feature:VolumeSnapshotDataSource]`](https://github.com/kubernetes/kubernetes/blob/2bbc0e3c54fdf11b0006efe7f65f6a416ea15d32/test/e2e/storage/testsuites/provisioning.go#L200)
* [`should provision storage with pvc data source`](https://github.com/kubernetes/kubernetes/blob/2bbc0e3c54fdf11b0006efe7f65f6a416ea15d32/test/e2e/storage/testsuites/provisioning.go#L238)

Neither of them are marked as `[LinuxOnly]`, therefore I think they should work on Windows.